Add install path for generated plugins

Without installation, local generated libraries have rpath set
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -122,6 +122,7 @@
     add_library(freshwrapper-pepperflash SHARED ${COMMON_OBJ_LIST} config_pepperflash.c)
     target_link_libraries(freshwrapper-pepperflash ${REQ_LIBRARIES})
     target_link_libraries(dep_check freshwrapper-pepperflash)
+    install(TARGETS freshwrapper-pepperflash LIBRARY DESTINATION lib/mozilla/plugins/)
 endif()
 
 # wrapper for Native Client
@@ -130,6 +131,7 @@
     add_library(freshwrapper-nacl SHARED ${COMMON_OBJ_LIST} config_nacl.c)
     target_link_libraries(freshwrapper-nacl ${REQ_LIBRARIES})
     target_link_libraries(dep_check freshwrapper-nacl)
+    install(TARGETS freshwrapper-nacl LIBRARY DESTINATION lib/mozilla/plugins/)
 endif()
 
 # libpdf wrapper
@@ -138,4 +140,5 @@
     add_library(freshwrapper-libpdf SHARED ${COMMON_OBJ_LIST} config_libpdf.c)
     target_link_libraries(freshwrapper-libpdf ${REQ_LIBRARIES})
     target_link_libraries(dep_check freshwrapper-libpdf)
+    install(TARGETS freshwrapper-libpdf LIBRARY DESTINATION lib/mozilla/plugins/)
 endif()
